.github/workflows/test.yaml

```diff
@@ -4,6 +4,30 @@ on:
   workflow_call:
 
 jobs:
+  inclusive-naming-check:
+    name: Inclusive naming
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: canonical-web-and-design/Inclusive-naming
```

Comment:
Shouldn't this be https://github.com/canonical/Inclusive-naming ?

Reply:
Indeed